DynamoDB/CreateTable.js

```diff
@@ -13,25 +13,25 @@ var dynamodb = new AWS.DynamoDB();
 
 var params = {
     TableName : "V-Transfer",
-    KeySchema: [       
+    KeySchema: [
         { AttributeName: "PK", KeyType: "HASH"},  //Partition key
         { AttributeName: "SK", KeyType: "RANGE" }  //Sort key
     ],
-    AttributeDefinitions: [       
+    AttributeDefinitions: [
         { AttributeName: "PK", AttributeType: "S" },
         { AttributeName: "SK", AttributeType: "S" },
-        { AttributeName: "U_ID", AttributeType: "S" },
+        { AttributeName: "GS1_PK", AttributeType: "S" },
         { AttributeName: "LS1_SK", AttributeType: "S"}
     ],
-    ProvisionedThroughput: {       
-        ReadCapacityUnits: 10, 
+    ProvisionedThroughput: {
+        ReadCapacityUnits: 10,
         WriteCapacityUnits: 10
     },
     GlobalSecondaryIndexes: [{
-        IndexName: "FIND_FILE",
+        IndexName: "FIND_FILE_BY_URL",
         KeySchema: [
             {
-                AttributeName: "U_ID",
+                AttributeName: "GS1_PK",
```

Comment:
where is SK??

Reply:
SK is the table's sort key.

Reply:
For GSI "FIND_FILE_BY_URL" we don't need to define a composite index. Only PK can work for us.

Reply:
For more information refer this [https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GSI.html](url)

Reply:
For more information, you can refer to this  [https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GSI.html](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GSI.html)